static/app/views/dashboardsV2/widgetCard.tsx

```diff
@@ -211,36 +246,13 @@ class WidgetCard extends React.Component<Props> {
             <WidgetTitle>{widget.title}</WidgetTitle>
             {this.renderContextMenu()}
           </WidgetHeader>
-          <LazyLoad once height={200}>
-            <WidgetQueries
-              api={api}
-              organization={organization}
-              widget={widget}
-              selection={selection}
-            >
-              {({tableResults, timeseriesResults, errorMessage, loading}) => {
-                return (
-                  <React.Fragment>
-                    {typeof renderErrorMessage === 'function'
-                      ? renderErrorMessage(errorMessage)
-                      : null}
-                    <WidgetCardChart
-                      timeseriesResults={timeseriesResults}
-                      tableResults={tableResults}
-                      errorMessage={errorMessage}
-                      loading={loading}
-                      location={location}
-                      widget={widget}
-                      selection={selection}
-                      router={router}
-                      organization={organization}
-                    />
-                    {this.renderToolbar()}
-                  </React.Fragment>
-                );
-              }}
-            </WidgetQueries>
-          </LazyLoad>
+          {noLazyLoad ? (
+            this.renderChart()
+          ) : (
+            <LazyLoad once resize height={200}>
```

Comment:
the old LazyLoad component didn't have `resize` prop, is that still supposed to be the case?

Reply:
Hey Shruthi, there's an edge case when a user resizes their window to a larger size, any new widgets that come into view don't lazy load. 
ie users window only shows 3 widgets, when they make their window bigger they see 6 widgets but the 3 new widgets are blank and don't lazy load.
Adding `resize` fixes this.